### PR TITLE
Shutdown

### PR DESCRIPTION
**Pull request by g4s8**
#297 - shutdown take and deploy command in `.rultor.yml` + `@todo`
(#440 PR)

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#456](https://github.com/zerocracy/farm/issues/456) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#456](https://github.com/zerocracy/farm/issues/456) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by fabriciofx** [comment]
@yegor256 This PR seems fine. I accept it.

File: src/main/java/com/zerocracy/tk/TkShutdown.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import org.takes.Response;
+import org.takes.Take;
+import org.takes.rs.RsEmpty;
+
+/**
+ * Shutdown the app.
+ *
+ * @author Kirill (g4s8.public@gmail.com)
+ * @version $Id$
+ * @since 0.20
```
Comment (fabriciofx) [inline src/main/java/com/zerocracy/tk/TkShutdown.java]: @g4s8 Let's change to next release (0.21).

File: src/test/java/com/zerocracy/tk/TkShutdownTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import org.junit.Test;
+import org.takes.facets.hamcrest.HmRsStatus;
+import org.takes.rq.RqFake;
+
+/**
+ * Test case for {@link TkShutdown}.
+ *
+ * @author Kirill (g4s8.public@gmail.com)
+ * @version $Id$
+ * @since 0.20
```
Comment (fabriciofx) [inline src/test/java/com/zerocracy/tk/TkShutdownTest.java]: @g4s8 The same above. Next release.

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was successfully finished: +15 points just awarded to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), total is [+330](https://www.0crat.com/u/fabriciofx)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+8300](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#456](https://github.com/zerocracy/farm/issues/456) is now out of scope

**Pull request merged**